content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Cut Service Mesh Overhead with Istio Ambient Mesh"
```

Comment:
I already gave my feedback in slack but will repeat here: I think the general idea is good but the punchline really downplays the result. Saving $23 is... meh.

Doing a real-er world test like deploying bookinfo and/or micro-services demo in 10-100 namespaces should bump that up substantially and give more realistic numbers.

It may also be nice to compare "sidecar (naive users)", "sidecar with Sidecar (real prod users @ scale)", "ambient".

Comparing w/o `Sidecar` can easily give absurd numbers - I shared my own tests giving $50/mo vs $500k/mo, but its not really realistic to what a real users spend would be.

Reply:
I agree with your comment, which is very good. I think Greg had issues with bookinfo handling requests during his test. Our intention is to publish this as it is based on small env test (which users can produce if they want to) and have a follow up blog about a more realistic app (microservices app you pointed out) on larger cluster.  The reason is timing - to rework/rerun could take much longer, would be nice to get this out before alpha goes out to encourage folks to use alpha. 

While $23 is not a lot giving the small scale env, if you consider percentage of saving it is HUGE (over 90%) already.

Reply:
No where do we make any claims that the percentage saving scales beyond a demo. Most things (not from Istio specifically, just in the industry) that are showing off in a tiny demo _don't_ actually translate to benefits at scale. If I read this I would not be impressed at all, TBH. 

I don't think publishing this as-is is beneficial. If that mean we publish months from now, that seems ok to me.

Reply:
I think the above feedback is resolved by running with a more realistic app (online boutique).  Please reopen if you feel differently.